<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>724</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="satPasses">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Satellite Passes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Satellite</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="11">
         <widget class="QTableWidget" name="satPassesTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>startTime</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>startAzimuth</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>maxElevation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>endAzimuth</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>endTime</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>totalTime</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="1" colspan="4">
         <widget class="QLabel" name="coordLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>AzEl</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="4">
         <widget class="QLabel" name="azElLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Coordinates</string>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QToolButton" name="tleOK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8" colspan="2">
         <widget class="QComboBox" name="tleSelect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="6">
         <widget class="QComboBox" name="satBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="satFreq">
       <attribute name="title">
        <string>Satellite Frequencies</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="satFreqTable">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mode</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Downlink</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Uplink</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Inverted</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="map">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QToolButton" name="mapOK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="mapSelect"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGraphicsView" name="mapView">
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="globe">
       <attribute name="title">
        <string>Globe</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="Marble::MarbleWidget" name="MarbleWidget" native="true">
          <property name="toolTip">
           <string>The Earth</string>
          </property>
          <property name="whatsThis">
           <string>The Marble widget displays a virtual globe.</string>
          </property>
          <property name="mapThemeId" stdset="0">
           <string>earth/openstreetmap/openstreetmap.dgml</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Latitude</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Longitude</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="latSecond">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>&quot;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QComboBox" name="longChoose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="longSecond">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>&quot;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="longHour">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="heightSpinBox"/>
        </item>
        <item row="0" column="5">
         <widget class="QToolButton" name="OK">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="longMinute">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>'</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="latHour">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="latMinute">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>'</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QComboBox" name="latChoose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="2" colspan="6">
         <widget class="QTextEdit" name="links"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSatMap">
    <property name="title">
     <string>SatMap</string>
    </property>
    <addaction name="actionUpdate_TLEs"/>
    <addaction name="actionDownload_map"/>
    <addaction name="actionUpdate_Freq"/>
    <addaction name="actionRefresh_TLEs"/>
    <addaction name="actionRefresh_maps"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSatMap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpdate_TLEs">
   <property name="text">
    <string>Download TLEs</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRefresh_TLEs">
   <property name="text">
    <string>Refresh TLEs</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDownload_map">
   <property name="text">
    <string>Download map</string>
   </property>
  </action>
  <action name="actionRefresh_maps">
   <property name="text">
    <string>Refresh maps</string>
   </property>
  </action>
  <action name="actionUpdate_Freq">
   <property name="text">
    <string>Update Freq</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Marble::MarbleWidget</class>
   <extends>QWidget</extends>
   <header>MarbleWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
